controllers/student.js

```diff
@@ -0,0 +1,5 @@
+const registerStudent = (req, res, next) => {
```

Comment:
Don't add them in separate files. Keep these inside routes files only. We can keep the body of function inside their respective routes.
`router.get('/students',(req, res,next)=>{ res.send({}).status(200)})`

Reply:
But I think when we have many controllers it will be very messy if we mix up all controllers in route files only.